doc/devel/testing.rst

```diff
@@ -144,13 +144,68 @@ tests it::
        fig, ax = plt.subplots()
        ax.plot(range(10), linestyle=(0, (3, 3)), lw=5)
 
-The first time this test is run, there will be no baseline image to compare
-against, so the test will fail.  Copy the output images (in this case
-:file:`result_images/test_lines/test_line_dashes.png`) to the correct
-subdirectory of :file:`baseline_images` tree in the source directory (in this
-case :file:`lib/matplotlib/tests/baseline_images/test_lines`).  Put this new
-file under source code revision control (with ``git add``).  When rerunning
-the tests, they should now pass.
+Baseline image generation
+-------------------------
+The first time this test is run, there will be no baseline image to compare against, so the test will fail. The baseline
+images would be generated either by installing ``matplotlib-baseline-images`` package or by running the baseline image
+generation command from the commandline. These methods are described below. Later on, developer can run ::
+
+    python3 -pytest
+
+and the tests will pass.
+
+
+How to install matplotlib-baseline-images package?
+--------------------------------------------------
+Developer can install the ``matplotlib-baseline-images`` package by running the command ::
+
+    python3 -mpip install  -ve  sub-wheels/matplotlib-baseline-images
```

Comment:
This should go in the developer set up guide (https://matplotlib.org/devel/contributing.html#installing-matplotlib-in-developer-mode) because this is something people need to know to get started with development now!